dace/config_schema.yml

```diff
@@ -509,6 +509,11 @@ required:
                         default: false
                         title: Verilator Enable Debug
                         description: Enable/disable verbose internal state debug output.
+                    part:
+                        type: str
+                        default: xcu280-fsvh2892-2L-e
+                        title: Target part
+                        description: Target part for the Xilinx Vivado project
```

Comment:
Ideally we can infer this from the Vitis shell. Do you think there's a way to do this? (apart from compiling something with Vitis and seeing which part is uses...)

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/5fb01f11d8799b01731ba8a0e1b08196135592a4